python/paddle/fluid/contrib/mixed_precision/fp16_utils.py

```diff
@@ -130,6 +130,13 @@ def _insert_cast_op(block, op, idx, src_dtype, dest_dtype):
 
 
 def find_true_prev_op(ops, var_name):
+    """
+    Find the true prev op that outputs var_name variable.
+
+    Args:
+        ops (list): A list of ops.
+        var_name (string): Variable name.
+    """
```

Comment:
I think we should assert that there should be only one op which generates this var.

Reply:
Just for safe!

Reply:
I'll fix it